docs/contributing.md

```diff
@@ -82,3 +102,8 @@ something that is confusing or not covered at all, feel free to
 [rd]: https://docs.radicle.xyz
 [rt]: https://github.com/radicle-dev/radicle-upstream/issues/1958
 [up]: http://docs.radicle.xyz/docs/using-radicle/creating-patches
+[ds]: https://discord.gg/HRdnwAwGbG
```

Comment:
This shows me an invite when I click it. Should it not point to the channel?
Like: https://discord.com/channels/841318878125490186/843873418205331506.

Reply:
🤷‍♂️ That's what we link to from our website?

Reply:
I will make another issue for this btw.

Reply:
I think you mixed up the links. You can delete this line and update the link in the text to `[Discord server][dc]`.